configure.ac

```diff
@@ -99,6 +99,17 @@ AM_CONDITIONAL([CRAY], [test `env | grep CRAY | wc -l` -gt 0])
 # Define an AM_CONDITIONAL to determine if you are on an Apple/Mac OS X
 AM_CONDITIONAL([APPLE], [test "x$target_vendor" = "xapple"])
 
+# Find which mpi launcher to use
+if test `command -v srun | grep -in "srun" | wc -l` -eq 1; then
```

Comment:
Autoconf already has a function to do this check: (`AC_CHECK_PROGS`)[https://www.gnu.org/software/autoconf/manual/autoconf-2.68/html_node/Generic-Programs.html).  Something like the following may work.

```
AC_CHECK_PROGS([MPI_LAUNCHER],[srun aprun mpirun])
```

Also, since some of the tests may need to oversubscribe the MPI processes, you may also want to use the `AC_PATH_PROGS_FEATURE_CHECK` to see if a flag is needed to do this (as it is for the OpenMPI `mpirun` command).